db/dummyData/insertDummySchemaData.js

```diff
@@ -0,0 +1,103 @@
+var mongoose = require('mongoose');
+const { mongoURL } = require('../config');
+const { Channel, User, Message } = require('../model/channel.js');
+
+const insertDummyData = async () => {
+  // make a connection
```

Comment:
```suggestion
  // Make a connection.
```

Please fix grammar in the rest of your comments.